ext_emconf.php

```diff
@@ -7,13 +7,13 @@
     'category' => 'plugin',
     'constraints' => [
         'depends' => [
-            'typo3' => '9.5.0-10.4.99',
+            'typo3' => '11.5.0-11.5.99',
```

Comment:
Next release should support 10.4 and 11.5.
Your changes seems to work on 10.4.
```suggestion
            'typo3' => '10.4.0-11.5.99',
```

Reply:
Updated ext_emconf.php